### PR TITLE
Improve Privacy and Security section

### PR DESCRIPTION
**Pull request by koobs**
Wordsmith section to improve clarity and provide additional information
related to the projects privacy position and with respect to the extensions
permission use.

References: #271

### CODE REVIEW

**Comment by devmount** [comment]
Thank you for providing proper English descriptions, this is such a great help 🙏🏻 I just have two minor questions before merging this PR (see above).

**Comment by devmount** [comment]
> I just have two minor questions before merging this PR (see above).

@koobs I'm sorry, I totally forgot to finish the review 🙈 My comments are visible now!

File: README.md [hunk 1]
```diff
@@ -50,7 +50,16 @@ Here is how ThirdStats looks like on the Thunderbird default dark theme and ligh
 
 ## Privacy and Security
 
-ThirdStats is fully contained and doesn't contact any third-party CDN servers. It runs locally without any webserver. ThirdStats stores processed data in a cache, but you can clear and disable it. ThirdStats will never store this data elsewhere, nor sent or sell it anywhere. ThirdStats needs permissions to access accounts, read email headers and download export files. ThirdStats will always seek to minimise use of permissions where possible, and use granular permissions where available. Thunderbirds Permissions API is currently limited to access the whole email instead of the email header only.
+ThirdStats runs entirely locally and will never contact, send or sell data to, any third-parties. 
+
+ThirdStats features require [Thunderbird permissions](https://developer.thunderbird.net/add-ons/mailextensions/supported-webextension-api) to function, but will always seeks to minimise permissions where possible, and use granular permissions where available [1]:
```
[the remark this answers is not in the document]
Comment (koobs) [reply README.md:55]: Both are correct usages. I'm Australian, so wrote 'minimise' (unconsciously).

 * Australian/British: `Minimise`
 * American: `Minimize`

Choices:

 1. Pick the LANG with which all **other documentation is already written and is consistent** (presumably your lang)
 2. Pick the LANG that is **most appropriate for the majority audiences** (America? Rest of world?)
 3. Roll a dice.

File: README.md [hunk 2]
```diff
@@ -50,7 +50,16 @@ Here is how ThirdStats looks like on the Thunderbird default dark theme and ligh
...
+
+ * Accounts: Access accounts and identities (read-only)
+ * Downloads: Export and download data as a file
+ * Messages: Access messages to create statistics (read-only)
+ * Storage: Store processed data (cache) for performance. Can be cleared or disabled at any time.
```
[the remark this answers is not in the document]
Comment (koobs) [reply README.md:60]: ### Goals/Considerations:

* Prefer to be *explicit rather than implicit* (gooin **above and beyond what Thunderbird does** where possible)
* Explicit **tie and describe** 'Permission' name with **Actual and specific addon function** that needs it'. eg: `Permission: <Feature Description>`

### Questions


1) Is `Storage` *required* to enable/store the cache?
2) Can we 'opt out' of `Storage` ? If so, we'd disable the 'Cache' option with a `(i)` indicator saying why and how to enable

tldr: If storage is *required* by *any* thirdstats function/feature, *listed by thunderbird or not*, describe the permission and the feature in docs.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (devmount) [reply README.md:55]: I see thanks! I'm bad at languages, maybe someone has time bring some consistency here, but for me (as I'm not that good in languages in general) it would be sufficient to have a correct English word at all 😅 So I'll just keep it the way you proposed 👍🏻

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (devmount) [reply README.md:60]: Wow always a pleasure to read your comments highlighting the problem with a minimal amount of words 😍 Totally agree with your proposed goals and will merge this PR as it is 👍🏻 

1. Yes, `Storage` is required to use the extension storage (= ThirdStats cache) at all
2. We can't afaik. Permissions are given by the user on installing the add-on and cannot be revoked after.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (koobs) [reply README.md:55]: Your English is perfectly fine my friend :)
